spec/lib/danger/ci_sources/bitrise_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "danger/ci_source/xcode_server"
+
+describe Danger::XcodeServer do
```

Comment:
You can make your terminal auto run the tests as you make changes by using `bundle exec guard`